Added font-face (#28532)
freeCodeCamp/freeCodeCamp: guide/english/css/fonts/index.md

In the above example, "Times New Roman" is the <family-name> of the font, while "serif" is the <generic-name>. Generic names are used as a fallback
mechanism for preserving style if the family-name is unavailable. A generic name should always be the last item in the list of font family names. Generic
family names are serif, sans-serif, monospace, cursive, fantasy, system-ui.
    

> It is also possible to use your own font wich is probably not installed on the system of the user.

Reviewer: ```suggestion
It is also possible to use your own font which may not be installed on the system of the user.
```

> To import this font, use the following code:

Reviewer: ```suggestion
To import fonts, the following format is used:
```
